frontend/package.json

```diff
@@ -12,12 +12,13 @@
     "html-webpack-plugin": "2.28.0",
     "http-proxy-middleware": "0.17.4",
     "node-sass": "4.5.2",
+    "prettier": "^1.4.4",
     "react-hot-loader": "3.0.0-beta.7",
     "sass-loader": "6.0.5",
     "style-loader": "0.17.0",
     "ts-loader": "2.0.3",
-    "tslint": "5.2.0",
-    "tslint-loader": "3.5.3",
+    "tslint": "^5.2.0",
```

Comment:
Ok, then this needs pinning too. At the moment only backend properties are pinned.